src/components/json/worldData.js

```diff
@@ -66,9 +68,42 @@ const data = [
 		bio: "Progate Community Intern",
 		url: "https://dsc.community.dev/bhagalpur-college-of-engineering/",
 		chapterName: "Indian Institute of Information Technology, Bhagalpur",
-		github: "https://github.com/helios027",
+		github: "http://github.com/helios027",
```

Comment:
Shouldn't this be `https`?

Reply:
@abhinavsri360 Yes, that change was made later in `main`. I'll update this PR. One moment.